### PR TITLE
Fix no sound bug

### PR DESCRIPTION
**Pull request by Rui-Minetest**
#5967

### CODE REVIEW

**Comment by Rui-Minetest** [comment]
@Fixer-007

**Comment by Rui-Minetest** [comment]
@Fixer-007

**Comment by Fixer-007** [comment]
Will try

**Comment by Rui-Minetest** [comment]
Updated.

File: src/itemdef.cpp [hunk 1]
```diff
@@ -167,7 +170,7 @@ void ItemDefinition::deSerialize(std::istream &is)
 
 	// Deserialize
 	int version = readU8(is);
-	if(version < 1 || version > 3)
+	if(version < 1 || version > 4)
```
Comment (nerzhul) [inline src/itemdef.cpp]: missing space

**Comment by nerzhul** [comment]
@Fixer-007 can you re-test it ?

**Comment by Fixer-007** [comment]
Will retest

File: src/itemdef.cpp (hunk 1 above)
Comment (Rui-Minetest) [inline src/itemdef.cpp]: Updated.

**Comment by Rui-Minetest** [comment]
Waiting for approval.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
Sorry, i should not have approved this, i don't understand the implementation enough to do so.

**Pull request merged**